google/cloud/spanner/instance_admin_connection.cc

```diff
@@ -62,38 +62,59 @@ class InstanceAdminConnectionImpl : public InstanceAdminConnection {
 
   future<StatusOr<gcsa::Instance>> CreateInstance(
       CreateInstanceParams p) override {
-    auto operation = RetryLoop(
-        retry_policy_prototype_->clone(), backoff_policy_prototype_->clone(),
-        Idempotency::kNonIdempotent,
-        [this](grpc::ClientContext& context,
-               gcsa::CreateInstanceRequest const& request) {
-          return stub_->CreateInstance(context, request);
+    auto stub = stub_;
+    return google::cloud::internal::AsyncLongRunningOperation<gcsa::Instance>(
+        background_threads_->cq(), p.request,
```

Comment:
consider: `std::move(p.request)`

Reply:
Done.

Reply:
I think the same trick may be used in the other functions too?

Reply:
I can't see anywhere else it would apply.